### PR TITLE
feat: [PIE-4903]: pipelines ux minor changes and sticky header

### PR DESCRIPTION
**Pull request by udayvunnam**
### Summary

- pipelines UX minor changes, executor alignment
- sticky header

#### Screenshots
image

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Failed_** 
 


            
 #### 2 Issues 
* Code issue 2 Code issues 

 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/10814)

File: src/modules/70-pipeline/pages/pipeline-list/PipelineListPage.module.scss [hunk 1]
```diff
@@ -14,6 +14,14 @@
   --page-header-height: 144px;
 }
 
+.subHeader{
```
Comment (rameshx) [inline src/modules/70-pipeline/pages/pipeline-list/PipelineListPage.module.scss:17]:
```suggestion
.subHeader {
```

File: src/modules/70-pipeline/pages/pipeline-list/PipelineListTable/PipelineListCells.tsx [hunk 2]
```diff
@@ -365,30 +369,32 @@ export const RecentExecutionsCell: CellType = ({ row }) => {
...
-                  }
-                />
-                <LabeValue
-                  label={getString('common.triggerName')}
-                  value={getString(mapTriggerTypeToStringID(i.executorInfo?.triggerType))}
-                />
-              </>
-            )}
-          </Layout.Vertical>
+          <div>
```
Comment (rameshx) [inline src/modules/70-pipeline/pages/pipeline-list/PipelineListTable/PipelineListCells.tsx:372]: Any reason for adding this outer `div`?

File: src/modules/70-pipeline/pages/pipeline-list/PipelineListTable/PipelineListTable.module.scss [hunk 3]
```diff
@@ -14,20 +14,38 @@
...
+
+.neverRan {
+  background-color: var(--grey-100);
+}
+
+.timeAgo{
+  margin-top: var(--spacing-xsmall) !important;
+}
+
+.icon{
```
Comment (rameshx) [inline src/modules/70-pipeline/pages/pipeline-list/PipelineListTable/PipelineListTable.module.scss:41]:
```suggestion
.icon {
```

File: src/modules/70-pipeline/pages/pipeline-list/PipelineListTable/PipelineListTable.module.scss [hunk 4]
```diff
@@ -14,20 +14,38 @@
...
+  text-transform: uppercase;
+  font-size: var(--spacing-4);
+  font-weight: 500;
+}
+
+.neverRan {
+  background-color: var(--grey-100);
+}
+
+.timeAgo{
```
Comment (rameshx) [inline src/modules/70-pipeline/pages/pipeline-list/PipelineListTable/PipelineListTable.module.scss:37]:
```suggestion
.timeAgo {
```

**Review by AshwinBhatkal-Harness — DISMISSED** [review]

**Review by rameshx — APPROVED** [review]

**Review by AshwinBhatkal-Harness — APPROVED** [review]

**Pull request merged**